iphone/metadata/de-DE/description.txt

```diff
@@ -1,40 +1,39 @@
-• Detaillierte Offline-Karten mit Orten, die dank OpenStreetMap.org auf anderen Karten nicht zu finden sind
-• Radwege, Wanderwege und Spazierwege
-• Höhenlinien, Höhenprofile, Gipfel und Steigungen
-• Navigation mit Abbiegehinweisen für Fußgänger, Radfahrer und Autos mit Sprachführung
-• Schnelle Offlinesuche auf der Karte und in den Lesezeichen
-• Import und Export von Lesezeichen in den Formaten KML/KMZ (GPX ist geplant)
-• Dunkles Design zum besseren Schutz Ihrer Augen
-• Länder und Regionen nehmen nicht viel Speicherplatz in Anspruch
-• Kostenlos und Open-Source
+• Detaillierte Offline-Karten mit Orten, die es auf anderen Karten nicht gibt
+• Rad-, Wander- und Spazierwege
+• Höhenlinien, Höhenprofile, Bergspitzen und Steigungen
+• Schritt-für-Schritt-Navigation für Fußgänger, Radfahrer und Autos mit Sprachführung
+• Schnelle Offlinesuche
+• Export und Import von Lesezeichen im KML/KMZ-Format (GPX ist geplant)
+• Dunkler Modus zum Schutz deiner Augen
+• … und viele weitere Funktionen!
 
 Organic Maps ist rein, organisch und mit Liebe gemacht:
 
-• Respektiert Ihre Privatsphäre
-• Schont Ihren Akku
-• Keine unerwarteten Kosten für mobile Daten
+• Respektiert deine Privatsphäre
+• Schont deinen Akku
+• Keine unerwarteten Gebühren für mobile Daten
 
-Frei von Trackern und sonstigem schädlichen Inhalt:
+Frei von Trackern und schlechten Dingen:
 
 • Keine Werbung
 • Kein Tracking
 • Keine Datensammlung
-• Kein "Telefonieren nach Hause"
+• Kein Telefonieren nach Hause
```

Comment:
I think we should keep the quotes, because it is not obvious what "Kein Telefonieren nach Hause" means. 
_Can we no longer make calls to our home?_

We should find a better term for it in the future.

Reply:
```suggestion
• Keine Telemetrie
```